release_notes.txt

```diff
@@ -1,3 +1,6 @@
+1.0.18
+-

```

Comment:
@mstein-ck Empty !?

Reply:
Yes, no changes were made for the new version. It's just a place to add the new release notes